drivers/tee/tee_core.c

```diff
@@ -6,18 +6,28 @@
 #define pr_fmt(fmt) "%s: " fmt, __func__
 
 #include <linux/cdev.h>
+#include <linux/cred.h>
 #include <linux/fs.h>
 #include <linux/idr.h>
 #include <linux/module.h>
 #include <linux/slab.h>
 #include <linux/tee_drv.h>
 #include <linux/uaccess.h>
+#include <crypto/hash.h>
+#include <crypto/sha.h>
 #include "tee_private.h"
 
 #define TEE_NUM_DEVICES	32
 
 #define TEE_IOCTL_PARAM_SIZE(x) (sizeof(struct tee_param) * (x))
 
+#define TEE_UUID_NS_NAME_SIZE	128
+
+/* TEE Client UUID name space identifier (UUIDv4) */
```

Comment:
How does this relate to UUIDv5? From where did you get these numbers?

Reply:
UUIDv5 needs to have UUID namespace identifier.

UUIDv4 is random and it was generated by my Linux kernel random number generation with uuidgen tool.

Reply:
So it's the namespace identifier for this UUID scheme, that is, the part left implementation defined by GP. That needs to be explained in a comment.

Reply:
Ok. will try to improve the wording.